doc/sources/installation/installation-rpi.rst

```diff
@@ -6,50 +6,134 @@ Installation on Raspberry Pi
 You can install Kivy manually (recommended), or you can download and boot KivyPie on the
 Raspberry Pi. Both options are described below.
 
+Raspberry Pi 4 manual installation (On Raspbian Buster)
+------------------------------------------------
+
+#. If you have installed Raspbian with a desktop, then you can skip to `Kivy dependencies`_.
```

Comment:
Can you please expand a little more here on what is meant by "with a desktop"? That is likely to be confusing to people who haven't thought about running in a terminal vs. desktop so they wouldn't know if this applies to them.